Makefile

```diff
@@ -171,7 +171,7 @@ test: manifests generate fmt vet envtest go-test.env go-test ## Run tests.
 go-test:
 ifeq ($(QUICK_BUILD),no)
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" \
-	go test $(TEST_FILES) -coverprofile $(COVER_PROFILE) $(TEST_OPTS)
+	go test $(TEST_FILES) -coverprofile $(COVER_PROFILE) $(TEST_OPTS) -v
```

Comment:
It is better to set "-v" when it is needed because it logs too much.
Like the following.
```
make go-test TEST_FILES=./internal/authproxy TEST_OPTS='-v -ginkgo.v'
```

Reply:
Understood.

But `make test` runs all test so I agree with smallest logs but `make go-test` is targeting specific test so I think when using `make go-test` we always need detailed test output.

VSCode Go extension also uses -v option by default for running package test. What do you think about?

Reply:
It is more convenient to show only the logs of the cases where errors occurred when iterating between fixes and tests.
If "-v" is specified, all logs that pass will also be shown. And too many logs exceed the buffer of the terminal, so you can't scroll back to see the top of the logs.

If "-v" is the default, it is better to make the `TEST_OPTS` variable default to "-v" so that it can be overridden.

Reply:
-v also show only the logs of the cases where errors occurred, right?
Also it can see the progress of each test suite and the number of test and so on.

```
coder@d1:cosmo-1$ make go-test TEST_FILES=github.com/cosmo-workspace/cosmo/internal/controllers
KUBEBUILDER_ASSETS="/home/coder/.local/share/kubebuilder-envtest/k8s/1.21.4-linux-amd64" \
go test github.com/cosmo-workspace/cosmo/internal/controllers -coverprofile cover.out  -v
=== RUN   Test_unstToObjectRef
=== RUN   Test_unstToObjectRef/OK
--- PASS: Test_unstToObjectRef (0.00s)
    --- PASS: Test_unstToObjectRef/OK (0.00s)
=== RUN   TestAPIs
Running Suite: Controller Suite - /projects/cosmo-1/internal/controllers
========================================================================
Random Seed: 1660227230

Will run 18 of 18 specs
••••••••••••••••••

Ran 18 of 18 Specs in 40.131 seconds
SUCCESS! -- 18 Passed | 0 Failed | 0 Pending | 0 Skipped
--- PASS: TestAPIs (40.14s)
PASS
coverage: 78.9% of statements
ok      github.com/cosmo-workspace/cosmo/internal/controllers   40.249s coverage: 78.9% of statements
```

without -v, it only show the coverage.

```
coder@d1:cosmo-1$ make go-test TEST_FILES=github.com/cosmo-workspace/cosmo/internal/controllers
KUBEBUILDER_ASSETS="/home/coder/.local/share/kubebuilder-envtest/k8s/1.21.4-linux-amd64" \
go test github.com/cosmo-workspace/cosmo/internal/controllers -coverprofile cover.out 
ok      github.com/cosmo-workspace/cosmo/internal/controllers   35.591s coverage: 78.7% of statements
```


> If "-v" is the default, it is better to make the TEST_OPTS variable default to "-v" so that it can be overridden.

Exactly. I will fix it, OK?

Reply:
I think we can create another variable. `GO_TEST_OPTS`

Reply:
BTW at the moment I revert the comment

Reply:
I was mistaken.
I think we can remove "go-test" from the premise and call "go test" directly from "make test".